Dockerfile

```diff
@@ -91,6 +91,12 @@ RUN apt-get update -q && \
     phpdismod pdo_pgsql && \
     phpdismod pgsql && \
     phpdismod xdebug && \
+    # Remove extra extensions installed via packages for other versions of PHP
+    rm -rf /usr/lib/php/20121212 && \
+    rm -rf /usr/lib/php/20131226 && \
+    # rm -rf /usr/lib/php/20151012 && \
```

Comment:
Got it, I thought 20160303 was the current version - maybe a comment saying: "20151012 is the version for PHP 7.0 / 7.1" would be helpful here?